
<launch>
  <group ns="stereo">
    <node pkg="stereo_image_proc" type="stereoproc"
          respawn="false" output="screen" name="stereoproc">
      <param name="do_rectify" type="bool" value="true"/>
      <param name="do_stereo" type="bool" value="True"/>
      <param name="do_calc_points" type="bool" value="True"/>

  	<remap from="raw_stereo" to="/stereo/raw_stereo_throttled" />
    </node>
  </group>
</launch>

